assets/app/scripts/app.js

```diff
@@ -194,6 +195,14 @@ angular
         templateUrl: 'views/browse/service.html',
         controller: 'ServiceController'
       })
+      .when('/project/:project/browse/storage', {
+        templateUrl: 'views/storage.html',
+        controller: 'StorageController'
+      })
+      .when('/project/:project/browse/persistentvolumeclaims/:pvc', {
```

Comment:
I know you mentioned code not ready, but figured I'd throw a thought or two in here. Would we want to keep these routes associated?

```javascript
.when('/project/:project/browse/storage'
.when('/project/:project/browse/storage/persistentvolumeclaims/:pvc', {
.when('/project/:project/browse/storage/persistentvolumeclaims/attach', {  
```
As we add routes I think it will pay off if we can keep patterns as consistent as possible.